File: app/src/main/java/com/example/sunddenfix/retrofit/viewIm/MainView.java
package com.example.sunddenfix.retrofit.viewIm;

import com.example.sunddenfix.retrofit.base.IView;
import com.example.sunddenfix.retrofit.bean.CountryModel;

import java.util.List;

/**
 * @author wangchengmeng
 */
public interface MainView extends IView {
    void getCountrySuccess(List<CountryModel> models, int status);

    void refreshOrDisLayout(int status);

    void loadNorMoreData();
}

File: app/src/main/java/com/example/sunddenfix/retrofit/viewIm/MainView.kt
package com.example.sunddenfix.retrofit.viewIm

import com.example.sunddenfix.retrofit.base.IView
import com.example.sunddenfix.retrofit.model.CountryModel

interface MainView : IView {

    fun updateText(model: CountryModel)

    fun updateText2(model: String)
}